### PR TITLE
Order Popup Fix

### PR DESCRIPTION
**Pull request by zans-laksa**
Fixed invalid request on order popup close.
Fixed some console errors.

### CODE REVIEW

File: packages/scandipwa/src/store/Order/Order.dispatcher.js [hunk 1]
```diff
@@ -28,6 +28,16 @@ export class OrderDispatcher {
             (error) => dispatch(showNotification('error', getErrorMessage(error)))
         );
     }
+
+    async getOrderById(dispatch, id) {
+        try {
+            const { getOrderById: result } = await fetchQuery(OrderQuery.getOrderByIdQuery(id));
+            return result;
```
Comment (IvansZuks) [inline packages/scandipwa/src/store/Order/Order.dispatcher.js:35]: missed white line before return

File: packages/scandipwa/src/store/Order/Order.dispatcher.js [hunk 2]
```diff
@@ -28,6 +28,16 @@ export class OrderDispatcher {
...
+        } catch (error) {
+            dispatch(showNotification('error', getErrorMessage(error)));
```
Comment (IvansZuks) [inline packages/scandipwa/src/store/Order/Order.dispatcher.js:37]: missed white line before return

**Review by IvansZuks — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/store/Order/Order.dispatcher.js (hunk 1 above)
Comment (zans-laksa) [reply packages/scandipwa/src/store/Order/Order.dispatcher.js:36]: Done.

File: packages/scandipwa/src/store/Order/Order.dispatcher.js (hunk 2 above)
Comment (zans-laksa) [reply packages/scandipwa/src/store/Order/Order.dispatcher.js:38]: Done.

**Review by IvansZuks — APPROVED** [review]

**Pull request merged**